modules/ipi-install-configuring-the-bios.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-the-bios_{context}"]
+= (Optional) Configuring the BIOS
+
+The following procedure configures the BIOS during the installation process.
+
+.Procedure
+. Create the manifests.
+
+. Modify the `BareMetalHost` resource file corresponding to the node:
++
+[source,terminal]
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-*.yaml
+----
+
+. Add the BIOS configuration to the `spec` section of the `BareMetalHost` resource:
++
+[source,terminal]
```

Comment:
Should this be source,yaml? (It is in the RAID module)

Reply:
Yes. Fixed.